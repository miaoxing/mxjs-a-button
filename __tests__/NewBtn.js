import {NewBtn} from '../';
import {MemoryRouter} from 'react-router';
import {render} from '@testing-library/react';

describe('NewBtn', () => {
  test('default', () => {
    const {container} = render(<MemoryRouter>
      <NewBtn href="foo">bar</NewBtn>
    </MemoryRouter>);

    expect(container).toMatchSnapshot();
  });

  test('text', () => {
    const {container} = render(<MemoryRouter>
      <NewBtn data-testid="foo" href="foo">bar</NewBtn>
    </MemoryRouter>);

    expect(container).toMatchSnapshot();
  });
});
