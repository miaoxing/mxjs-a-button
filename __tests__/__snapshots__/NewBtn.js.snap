// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewBtn default 1`] = `
<div>
  <a
    class="ant-btn ant-btn-primary"
    href="foo"
    tabindex="0"
  >
    <span>
      bar
    </span>
  </a>
</div>
`;

exports[`NewBtn text 1`] = `
<div>
  <a
    class="ant-btn ant-btn-primary"
    data-testid="foo"
    href="foo"
    tabindex="0"
  >
    <span>
      bar
    </span>
  </a>
</div>
`;
